examples/ogbn_products_revgnn.py

```diff
@@ -0,0 +1,278 @@
+import os.path as osp
+
+import torch
+import torch.nn.functional as F
+from ogb.nodeproppred import Evaluator, PygNodePropPredDataset
```

Comment:
Unknow frozen error due to the import order of ogb. This can be fixed by moving the import of ogb to the end. However, this fails to pass yapf and isort. Any clue about how to fix it? @rusty1s

` File "<frozen importlib._bootstrap>", line 1007, in _find_and_load
  File "<frozen importlib._bootstrap>", line 982, in _find_and_load_unlocked
  File "<frozen importlib._bootstrap>", line 925, in _find_spec
  File "/home/lig/miniconda3/envs/pyg_dev/lib/python3.9/site-packages/_distutils_hack/__init__.py", line 90, in find_spec
    return method()
  File "/home/lig/miniconda3/envs/pyg_dev/lib/python3.9/site-packages/_distutils_hack/__init__.py", line 101, in spec_for_distutils
    mod = importlib.import_module('setuptools._distutils')
  File "/home/lig/miniconda3/envs/pyg_dev/lib/python3.9/importlib/__init__.py", line 127, in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
  File "/home/lig/miniconda3/envs/pyg_dev/lib/python3.9/site-packages/setuptools/__init__.py", line 16, in <module>
    import setuptools.version
  File "/home/lig/miniconda3/envs/pyg_dev/lib/python3.9/site-packages/setuptools/version.py", line 1, in <module>
    import pkg_resources
  File "<frozen importlib._bootstrap>", line 211, in _lock_unlock_module
  File "<frozen importlib._bootstrap>", line 107, in acquire
KeyboardInterrupt`

Reply:
fixed.